fetch.js

```diff
@@ -40,14 +40,28 @@ function add_table_row(handle, json){
 async function do_magic(){
     ipcRenderer.on('get-it', (event,message) =>
     {
-        const array = message.split(',')
-        console.log(array, "here")
-        const handle1 = array[0]
-        const repo1 = array[1]
-        handle = handle1
-        repo = repo1
-        console.log(handle,repo)
-        fetch_data_for_single_user(handle)    
+        console.log("hello")
+        //const array = message.split(',')
```

Comment:
Why the comments? Why not remove these line? We can always get them back from previous commits.